package/skelcd-control-openSUSE.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed May  5 12:54:16 UTC 2021 - Guillaume GARDET <guillaume.gardet@opensuse.org>
+
+- Fix path of repo for Leap 15.3 armv7
```

Comment:
Please add a bugzilla number (`bsc#....`) for reference.

Reply:
Done